docs/content/posts/2023-02-23-release-1.15.4.md

```diff
@@ -0,0 +1,144 @@
+---
+authors:
+- danny: null
+  name: Danny Cranmer
+date: "2023-02-23T17:00:00Z"
+excerpt: The Apache Flink Community is pleased to announce a bug fix release for Flink
+  1.15.
+title: Apache Flink 1.15.4 Release Announcement
+---
+
+The Apache Flink Community is pleased to announce the fourth bug fix release of the Flink 1.15 series.
+
+This release includes 46 bug fixes, vulnerability fixes, and minor improvements for Flink 1.15.
```

Comment:
Checking the [JIRA list](https://issues.apache.org/jira/issues/?jql=project%20%3D%20FLINK%20AND%20fixVersion%20%3D%201.15.4), there're totally 49 issues with 1.15.4 as FixVersion, and 48 are resolved. Please check and update the list here.

Related, FLINK-31133 is still not resolved, and we need to confirm whether it should be excluded. I've also left a comment on the JIRA issue.

Reply:
> Checking the [JIRA list](https://issues.apache.org/jira/issues/?jql=project%20%3D%20FLINK%20AND%20fixVersion%20%3D%201.15.4), there're totally 49 issues with 1.15.4 as FixVersion, and 48 are resolved. Please check and update the list here.

(FLINK-31272)[https://issues.apache.org/jira/browse/FLINK-31272] and (FLINK-31286)[https://issues.apache.org/jira/browse/FLINK-31286] had been merged after the release branch was cut. I have bumped them to 1.15.5. Now I see [46](https://issues.apache.org/jira/projects/FLINK/versions/12352526) issues in the release again. FYI there is a step in the release process to check this later. 

(FLINK-31133)[https://issues.apache.org/jira/browse/FLINK-31133] has been punted to 1.15.5 already.